From 2b22f7dd8bbb50c3e5f2c018c15e30573afa82e0 Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Fri, 15 Dec 2023 18:25:47 +0100
Subject: [PATCH] packages: tvheadend: fix proxy

Fix proxy for tvheadend addon, change http root from default one to /tvheadend
This is second part for merge request !1121
---
 multimedia/tvheadend/files/tvheadend.init | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/multimedia/tvheadend/files/tvheadend.init b/multimedia/tvheadend/files/tvheadend.init
index 44e71c44a..fc3c9f990 100644
--- a/multimedia/tvheadend/files/tvheadend.init
+++ b/multimedia/tvheadend/files/tvheadend.init
@@ -67,7 +67,7 @@ load_uci_config() {
 	[ -n "$bindaddr" ] && procd_append_param command --bindaddr "$bindaddr"
 	config_get http_port server http_port
 	[ -n "$http_port" ] && procd_append_param command --http_port "$http_port"
-	config_get http_root server http_root
+	config_get http_root server http_root "/tvheadend"
 	[ -n "$http_root" ] && procd_append_param command --http_root "$http_root"
 	config_get htsp_port server htsp_port
 	[ -n "$htsp_port" ] && procd_append_param command --htsp_port "$htsp_port"
-- 
2.43.0

